index.html

```diff
@@ -65,5 +62,218 @@ <h1 class = "top-heading"> Hi there! I'm Sameed</h1>
+                    <ul class="highlights">
+                        <li class="project-type">CANOPY</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
+                        <li class="which-end">Back End Dev</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
+                        <li class="project-year">2015</li>
+                    </ul>
+                </div>
+                <div class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="tags">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <div class="project-button">
+                    <span class="project-text">
+                        See Project
+                    </span>
+                </div>
+            </div>
+        </article>
+
+        <article class="first-project">
+            <img src="./Images/4 multi-post stories.png" alt="" class="work-image">
+            <div class="details-container">
+                <div class="heading-and-highlights">
+                    <h1 class="project-heading">Multi-Post<br>Stories</h1>
+                    <ul class="highlights">
+                        <li class="project-type">CANOPY</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
+                        <li class="which-end">Back End Dev</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
+                        <li class="project-year">2015</li>
+                    </ul>
+                </div>
+                <div class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="tags">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <div class="project-button">
+                    <span class="project-text">
+                        See Project
+                    </span>
+                </div>
+            </div>
+        </article>
+
+    </section>
+
+    <section class="about-section">
+
+        <article class="about-myself">
+            <h1 class = "about-heading">About<br>Myself
+            </h1>
+            <p class="about-description">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] Kindly adjust the padding around your `about me ` section to match the original design and also adjust the border-radius on the top right corner

![image](https://user-images.githubusercontent.com/98453979/213705615-51952d3e-39d4-4ea7-8b6b-9bb7b1ddd96c.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213706553-1781d911-4b55-42a9-914f-9a5e5c37b2f5.png)